source/server/hot_restart.proto

```diff
@@ -17,12 +17,22 @@ message HotRestartMessage {
     }
     message Terminate {
     }
+    // A separate socket is established for forwarding undeliverable quic udp packets
+    // from the parent instance to the child instance.
+    message ForwardedUdpPacket {
+      string local_addr = 1;
+      string peer_addr = 2;
+      uint64 receive_time_epoch_microseconds = 3;
+      bytes payload = 4;
+      uint32 worker_index = 5;
+    }
     oneof request {
       PassListenSocket pass_listen_socket = 1;
       ShutdownAdmin shutdown_admin = 2;
       Stats stats = 3;
       DrainListeners drain_listeners = 4;
       Terminate terminate = 5;
+      ForwardedUdpPacket forwarded_udp_packet = 6;
```

Comment:
Given that these are sent on a separate UDS, I don't think this should be part of this oneof. I think it can be a new separate message. And maybe in it's own oneof (with only 1 entry for now) to allow future expansion. WDYT?

Reply:
Conceptually agree, but the whole existing mini-RPC framework relies on this being the message type, so using a different type would make for a bunch of duplicate code or significant changes to stuff I'm trying not to impact.